src/messages/MOSDOp.h

```diff
@@ -7,9 +7,9 @@
  *
  * This is free software; you can redistribute it and/or
  * modify it under the terms of the GNU Lesser General Public
- * License version 2.1, as published by the Free Software 
+ * License version 2.1, as published by the Free Software
  * Foundation.  See file COPYING.
- * 
+ *
  */
```

Comment:
Unnecessary whitespace changes

Reply:
I guess these came in from the commits of original PR. These are removed now.